packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -44,9 +45,14 @@ export interface SnapshotWithBlobs {
 /**
  * State saved by a container at close time, to be used to load a new instance
  * of the container to the same state
+ *
+ * This is very similar to {@link @fluidframework/protocol-definitions/internal#ISnapshot}, but the difference is
+ * that the blobs of ISnapshot of are ArrayBufferLike, while the blobs of this interface are serializable because
+ * they are already converted to string.
+ *
  * @internal
  */
-export interface IPendingContainerState extends SnapshotWithBlobs {
+export interface IPendingContainerState extends ISnapshotInfo {
```

Comment:
You're adding snapshotSequenceNumber to IPendingContainerState right? Is that needed?

Reply:
100%

Reply:
I need to know it for data virtualization purposes, and to maintain container correctness